### PR TITLE
fix: use latest version in configure&deploy for oathkeeper closes #701

### LINKED ISSUE #701: Update configure&deploy Oathkeeper guide
The [configure&deploy](https://www.ory.sh/docs/oathkeeper/configure-deploy) guide is using a outdated version, lets update it to use latest. 
 `git checkout {useLatestRelease('oathkeeper')}`
https://github.com/ory/docs/pull/677/files

we basically have to use a codeblock like this 
https://github.com/ory/docs/blob/11fd0ecaa2909813db37814da8821a77fc808c8e/src/components/Install/Docker.tsx
for the docker commands.

### PR DESCRIPTION
**Pull request by gen1us2k**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: docs/oathkeeper/configure-deploy.md [hunk 1]
```diff
@@ -235,23 +240,27 @@ creating such keys. All common JWT algorithms are supported (RS256, ES256,
 HS256, ...). Let's generate a key for the RS256 algorithm that will be used by
 the id_token mutator:
 
-```sh
-docker run oryd/oathkeeper:v0.38.16-beta.1 credentials generate --alg RS256 > jwks.json
+```mdx-code-block
+
```
Comment (vinckr) [inline docs/oathkeeper/configure-deploy.md:244]:
```suggestion
```

File: docs/oathkeeper/configure-deploy.md [hunk 2]
```diff
@@ -267,8 +276,11 @@ apply these best practices.
 Before building the Docker Image, we need to make sure that the local Ory
 Oathkeeper Docker Image is on the most recent version:
 
-```sh
-docker pull oryd/oathkeeper:v0.38.16-beta.1
+```mdx-code-block
+
```
Comment (vinckr) [inline docs/oathkeeper/configure-deploy.md:280]:
```suggestion
```

**Review by vinckr — CHANGES_REQUESTED** [review]
Great, the guide now works with no problems 🎉

File: docs/oathkeeper/configure-deploy.md [hunk 3]
```diff
@@ -107,12 +107,17 @@ Oathkeeper works. Let's define three rules:
...
    `https://httpbin.org/anything/id_token` using the `id_token` mutator.
 
-```shell
+```mdx-code-block
+
+import { useLatestRelease } from '@site/src/hooks'
+import CodeBlock from '@theme/CodeBlock'
+
+<CodeBlock className="language-shell">{`
 cat << EOF > rules.json
```
Comment (aeneasr) [inline docs/oathkeeper/configure-deploy.md:116]:
```suggestion
<CodeBlock className="language-shell">{`cat << EOF > rules.json
```

File: docs/oathkeeper/configure-deploy.md [hunk 4]
```diff
@@ -235,23 +240,26 @@ creating such keys. All common JWT algorithms are supported (RS256, ES256,
 HS256, ...). Let's generate a key for the RS256 algorithm that will be used by
 the id_token mutator:
 
-```sh
-docker run oryd/oathkeeper:v0.38.16-beta.1 credentials generate --alg RS256 > jwks.json
+```mdx-code-block
+<CodeBlock className="language-shell">{`
+docker run oryd/oathkeeper:${useLatestRelease('oathkeeper')} credentials generate --alg RS256 > jwks.json
+`}</CodeBlock>
```
Comment (aeneasr) [inline docs/oathkeeper/configure-deploy.md:246]:
```suggestion
<CodeBlock className="language-shell">{`docker run oryd/oathkeeper:${useLatestRelease('oathkeeper')} credentials generate --alg RS256 > jwks.json`}</CodeBlock>
```

File: docs/oathkeeper/configure-deploy.md [hunk 5]
```diff
@@ -267,8 +275,10 @@ apply these best practices.
 Before building the Docker Image, we need to make sure that the local Ory
 Oathkeeper Docker Image is on the most recent version:
 
-```sh
-docker pull oryd/oathkeeper:v0.38.16-beta.1
+```mdx-code-block
+<CodeBlock className="language-shell">{`
+docker pull oryd/oathkeeper:${useLatestRelease('oathkeeper')}
+`}</CodeBlock>
```
Comment (aeneasr) [inline docs/oathkeeper/configure-deploy.md:281]:
```suggestion
<CodeBlock className="language-shell">{`docker pull oryd/oathkeeper:${useLatestRelease('oathkeeper')}`}</CodeBlock>
```

**Review by aeneasr — DISMISSED** [review]
Awesome, the guide works great now! Just a few nitpicks. All newlines in code examples will show up in the rendered HTML as well which looks strange

Bildschirmfoto 2022-03-30 um 11 22 52

so we need to remove any unwanted newlines :)

Also there seems to be an image which doesnt work at the end of the file, could you maybe fix that in this pr too?

Bildschirmfoto 2022-03-30 um 11 22 33

**Comment by gen1us2k** [comment]
@aeneasr I added missing image

**Review by vinckr — APPROVED** [review]

**Pull request merged**